tests/main/snapd-homedirs-vendored/task.yaml

```diff
@@ -81,7 +81,7 @@ execute: |
     MATCH "^homedirs=/remote/users$" < /var/lib/snapd/system-params
 
     echo "And that the AppArmor tunable file is proper"
-    MATCH "^@{HOMEDIRS}\\+=\"/remote/users\"$" < /etc/apparmor.d/tunables/home.d/snapd
+    MATCH "^@{HOMEDIRS}\\+=\"/remote/users\" \"/home\"$" < /etc/apparmor.d/tunables/home.d/snapd
```

Comment:
So I'm a bit confused by why this is getting added here - the thing is that the default value of @{HOMEDIRS} is set by `/etc/apparmor.d/tunables/home` to `@{HOMEDIRS}=/home/`. Is there any issue in adding it twice?

Reply:
Thank you for the review!  I had to add `/home` in order for the regex to pass. Due to  the modifications requested in https://github.com/snapcore/snapd/pull/13145#discussion_r1461581688, `homedirs` will now include `/home` causing `configureHomedirsInAppArmorAndReload` to include `/home` in `@{HOMEDIRS}`.

Having `/home` defined multiple times in `@{HOMEDIRS}=` shouldn't cause any issues, however I don't mind reverting the behavior if needed.